docs/schemes/oauth2.md

```diff
@@ -14,6 +14,7 @@ auth: {
       authorization_endpoint: 'https://accounts.google.com/o/oauth2/auth',
       userinfo_endpoint: 'https://www.googleapis.com/oauth2/v3/userinfo',
       scope: ['openid', 'profile', 'email'],
+      access_type: 'online',
```

Comment:
Isn't better to leave default `undefined`? Making usage optional.

Reply:
Yes, I think that would be ok

Reply:
Hi, are there any further thoughts on this? It would be great if it could be merged as I am keen to upgrade my version of the module to benefit from other recent improvements.

In my opinion the concerns raised by @DevBrent are important but ultimately it is down to the developer to ensure their server side authorisation step handles the refresh tokens responsibly. Nuxt auth has support for refresh tokens generally, so why not support Google refresh token generation as well?

Reply:
Other than making it optional PR looks fine for merge

Reply:
Ok done (pardon the omitted word in the commit message)

Reply:
Whoops, I think you meant to actually update the code, not just the docs.
Not sure why the tests are failing, but all steps fail when I try to run the locally, i.e. different failures to those in circleci.